adapters/onebot/src/bot/message.ts

```diff
@@ -43,17 +44,28 @@ export class OneBotMessenger extends Messenger<BaseBot> {
     }
 
     // flush
-    if (!this.children.length) return
     const { type, author } = this.stack[0]
+    if (!this.children.length && !author.messageId) return
```

Comment:
swap these lines, so if there is an early return happened, the js engine no need to create the variables and destroy them immediately.


```suggestion
    if (!this.children.length && !author.messageId) return
    const { type, author } = this.stack[0]
```

Reply:
How can I get `author.messageId` if I do that?

Reply:
Oh, make sense.
It would be too wordy to write `this.stack[0].author`.